Validation/RecoVertex/python/plotting/vertexPlots.py

```diff
@@ -34,18 +34,18 @@
 _pvtagging = PlotGroup("pvtagging", [
     Plot("TruePVLocationIndexCumulative", xtitle="Signal PV status in reco collection", ytitle="Fraction of events", drawStyle="hist", normalizeToUnitArea=True, xbinlabels=["Not reconstructed", "Reco and identified", "Reco, not identified"], xbinlabelsize=15, xbinlabeloption="h", xgrid=False, ylog=True, ymin=1e-3, ratioCoverageXrange=[-0.5, 0.5]),
     Plot("TruePVLocationIndex", xtitle="Index of signal PV in reco collection", ytitle="Fraction of events", drawStyle="hist", normalizeToUnitArea=True, ylog=True, ymin=1e-5),
-    Plot("MisTagRate_vs_PU", xtitle="PU", ytitle="Mistag rate vs. PU", title="", xmax=_maxPU, ymax=_maxFake),
-    Plot("MisTagRate_vs_sum-pt2", xtitle="#Sigmap_{T}^{2}", ytitle="Mistag rate vs. #Sigmap_{T}^{2}", title="", xlog=True, ymax=_maxFake),
+    Plot("MisTagRate_vs_PU", xtitle="Number of simulated", ytitle="Mistag rate", title="", xmax=_maxPU, ymax=_maxFake),
```

Comment:
I think that there is a small typo here and the title of the x axis should be "Number of simulated interactions"?